### PR TITLE
3.6 genbindings abort on error

### PR DESCRIPTION
**Pull request by PatriceJiang**
Re: #

### Changelog

*

-------

### Continuous Integration

This pull request:

* [ ] needs automatic test cases check.
  > Manual trigger with `@cocos-robot run test cases` afterward.
* [ ] does not change any runtime related code or build configuration
  > If any reviewer thinks the CI checks are needed, please uncheck this option, then close and reopen the issue.

-------

### Compatibility Check

This pull request:

* [ ] changes public API, and have ensured backward compatibility with [deprecated features](https://github.com/cocos/cocos-engine/blob/v3.5.0/docs/contribution/deprecated-features.md).
* [ ] affects platform compatibility, e.g. system version, browser version, platform sdk version, platform toolchain, language version, hardware compatibility etc.
* [ ] affects file structure of the build package or build configuration which requires user project upgrade.
* [ ] **introduces breaking changes**, please list all changes, affected features and the scope of violation.

### CODE REVIEW

**Comment by PatriceJiang** [comment]
Consider merge in 3.6.2

File: native/tools/tojs/genbindings.py [hunk 1]
```diff
@@ -186,7 +186,11 @@ def generate (cfg, directory = None, section = None, filename = None):
             command = '%s -W ignore %s %s -s %s -t %s -o %s -n %s' % (python_bin, generator_py, cfg, section, target, directory, filename)
             print ("command : %s" % (command))
             # tasks.append(_run_cmd(command))
-            _run_cmd(command).communicate()
+            popen = _run_cmd(command)
+            popen.communicate()
+            if popen.returncode != 0:
+                print ("Error: falied to generate bindings for '%s'" % (section))
```
Comment (dumganhar) [inline native/tools/tojs/genbindings.py]: Typo: falied -> failed

Comment (PatriceJiang) [inline native/tools/tojs/genbindings.py]:
```suggestion
                print ("Error: failed to generate bindings for '%s'" % (section))
```

Comment (PatriceJiang) [reply native/tools/tojs/genbindings.py]: fixed

**Review by dumganhar — APPROVED** [review]

**Pull request merged**